src/explore-education-statistics-admin/src/pages/release/data/components/ReleaseDataUploadsSection.tsx

```diff
@@ -380,7 +380,13 @@ const ReleaseDataUploadsSection = ({
                       {block.isKeyStatistic && (
                         <p>
                           A key statistic associated with this data block will
-                          also be removed.
+                          be removed.
+                        </p>
+                      )}
+                      {block.isFeaturedTable && (
+                        <p>
+                          A featured table associated with this data block will
+                          be removed.
```

Comment:
As mentioned in the backend code, we should be able to provide a more informative message than this that could include the the featured table name. For example, something like: `The featured table "<featured table name>" is associated with this data block and will be removed`